### PR TITLE
Add companies owned by Shinsegae

### PR DESCRIPTION
**Pull request by seia-soto**
Gmarket is one of Amazon-like many-to-many ECommerce marketplace in Korea. I recently found that they're using onsite analytics during fixing breakage.

refs https://github.com/yous/YousList/pull/444

### CODE REVIEW

**Comment by philipp-classen** [comment]
According to Wikipedia, G-Market is owned by Shinsegae now:

https://en.wikipedia.org/wiki/G-Market

> Gmarket is an e-commerce website based in South Korea. The company was founded in 2000 as a subsidiary of Interpark, and was acquired by eBay in 2009, who subsequently sold it to Shinsegae at 3.4 trillion Korean Won.

**Comment by seia-soto** [comment]
> According to Wikipedia, G-Market is owned by Shinsegae now:
> 
> 
> 
> https://en.wikipedia.org/wiki/G-Market
> 
> 
> 
> > Gmarket is an e-commerce website based in South Korea. The company was founded in 2000 as a subsidiary of Interpark, and was acquired by eBay in 2009, who subsequently sold it to Shinsegae at 3.4 trillion Korean Won.

Confirmed. Please, keep this open. I'll add more websites owned by Shinsegae soon. I think the Shinsegae is the parent company of Gmarket now, and the Auction's parent company is Gmarket.

**Comment by seia-soto** [comment]
Referencing Korean wikipedia for wider view on subsidiaries.

**Comment by seia-soto** [comment]
Sites checked:

- https://www.shinsegae.com/index.do
- https://www.shinsegaegroupnewsroom.com/
- https://www.shinsegaepoint.com/
- https://store.emart.com/main/main.do
- http://www.traders.co.kr/index.jsp
- http://www.emarteveryday.co.kr/
- https://www.emart24.co.kr/
- https://www.shinsegaetvshopping.com/broadcast/main
- https://www.premiumoutlets.co.kr/rpage/integration/index
- https://www.starfield.co.kr/
- https://www.sikorea.co.kr/main
- http://www.shinsegaefood.com/main.sf
- https://www.shinsegae-lnb.com/html/main/index.html
- https://www.josunhotel.com/intro.do
- http://www.sinc.co.kr/main.do

File: db/patterns/shinsegae_dutyfree.eno [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ssgdfs.com
+applog.ssgdfs.com
+--- domains
+
+--- filters
+||applog.ssgdfs.com^
+||ssgdfs.com/kr/common/getPageIcfCd^
+--- filters
+
+ghostery_id: 3285
```
Comment (philipp-classen) [inline db/patterns/shinsegae_dutyfree.eno:16]: The id is not intended, right?

Comment (seia-soto) [reply db/patterns/shinsegae_dutyfree.eno:16]: Oh yes, it wasn't intended.

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**